features/apply_with_dependant_stacks.feature

```diff
@@ -0,0 +1,169 @@
+Feature: Apply command
```

Comment:
Typo in filename: `dependant` -> `dependent`

This typo appears several places in this PR (sorry to be pedantic!).